README.md

```diff
@@ -229,3 +230,10 @@ Many thanks to all the [Scribe-Data contributors](https://github.com/scribe-org/
   &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
   <br>
 </div>
+
+<a id="architecture-diagrams"></a>
+# Architecture Diagrams  [`⇧`](#contents)
+Scribe[[1]](https://en.wikipedia.org/wiki/Scribe) is a community of language enthusiasts, learners and developers creating tools to help people communicate with confidence.We strive to be an inclusive community, welcoming coders and open-source supporters of any level.  
+ Here is the overview of the [architecture diagrams](https://github.com/scribe-org/Organization/blob/main/ARCHITECTURE.md#full-architecture) of our organization, the services, the applications, the processes and the external systems and sources with which it interacts. 
+  It depicts the projects [Scribe-org](https://github.com/scribe-org) is developing and the relationship between the 
+  projects and external systems and sources.  
```

Comment:
Hey! :wave: 
Sorry - perhaps, I might have specified more in the issue description.

I was actually thinking the blurb about the architecture diagrams could just be at the very end of the introductory section of the `README.md`, i.e. right after the 

> Scribe applications are available on iOS...

That made sense to me, since the introduction section is already pointing out the other Scribe projects. Right after that, having the blurb about the architecture diagrams showing how the projects then connect kinda made sense logically to me. 

Two notes on the above:
- That would also eliminate the need to have to restate in the blurb what Scribe is as well.
- The other `README.md`s also follow a similar format for their introductory sections, which is why the above made sense to me also.